src/scales/scale.time.js

```diff
@@ -739,9 +740,12 @@ module.exports = Scale.extend({
 
 	getValueForPixel: function(pixel) {
 		var me = this;
+		var offsets = me._offsets;
 		var size = me._horizontal ? me.width : me.height;
-		var start = me._horizontal ? me.left : me.top;
-		var pos = (size ? (pixel - start) / size : 0) * (me._offsets.start + 1 + me._offsets.start) - me._offsets.end;
+		var offset = me.options.ticks.reverse ?
+			(me._horizontal ? me.right : me.bottom) - pixel :
+			pixel - (me._horizontal ? me.left : me.top);
+		var pos = (size ? offset / size : 0) * (offsets.start + 1 + offsets.end) - offsets.start;
```

Comment:
I'm wondering if we can simplify this formula. Does it ever actually happen that `size` is 0? If so, perhaps we could do `if (size <= 0) { return 0; }` first to keep this line from getting too complicated?

Reply:
I agree that a zero-check is not needed because no other scale is doing so. I just removed the check.